mypyc/ops.py

```diff
@@ -1161,6 +1161,38 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_unbox(self)
 
 
+class RaiseStandardError(RegisterOp):
+    """Raise built-in exception with an optional error string.
+
+    We have a separate opcode for this for convenience and to
+    generate smaller, more idiomatic C code.
+    """
```

Comment:
I don't love adding opcodes in general, but it does definitely result in smaller and better looking C code, so...